manifests/init.pp

```diff
@@ -46,6 +46,14 @@
     },
   }
 
+  if fact('os.release.major') == '8' {
+    exec { 'enable mariadb:10.5':
```

Comment:
The exec isn't necessary. The `package` type has a `dnfmodule` provider: https://www.puppet.com/docs/puppet/7/types/package.html#package-provider-dnfmodule

E.g. (untested):

```puppet
package { 'mariadb:10.5':
  ensure   => present,
  provider => dnfmodule,
  before   => ...,
}
```

Reply:
I see. I don't have a suitable RHEL8 system to test that on at the moment; will try it out when I do.

Reply:
The acceptance tests are running on EL8 & 9. E.g.: https://github.com/lsst-it/puppet-ccs_database/actions/runs/4378923295/jobs/7664275239

Reply:
OK; revised version passed the tests.